tests/back/Pim/Structure/EndToEnd/Attribute/ExternalApi/ListAttributeEndToEnd.php

```diff
@@ -144,6 +145,52 @@ public function testAttributeSearchByCode()
         $this->assertJsonStringEqualsJsonString($expected, $response->getContent());
     }
 
+    public function testAttributeSearchByUpdated()
+    {
+        /** @var Connection $connection */
+        $connection = $this->get('database_connection');
+        $affected = $connection->exec('UPDATE pim_catalog_attribute SET updated="2019-05-15 16:27:00" WHERE code="a_file"');
+        $this->assertEquals(1, $affected, 'There is more result as expected during test setup, the test will not work.');
```

Comment:
```suggestion
        Assert::assertEquals(1, $affected, 'There is more result as expected during test setup, the test will not work.');
```

Not a real issue here, no need to fix it, but for the next time :)  Static method `assertEquals` should not be called with `$this` (not possible with stricter languages as Java).

Reply:
Thanks for the explanation, Alexandre !